packages/trpc/server/profile-router/router.ts

```diff
@@ -200,7 +200,12 @@ export const profileRouter = router({
       try {
         const { email } = input;
 
-        return await sendConfirmationToken({ email });
+        await jobsClient.triggerJob({
+          name: 'send.confirmation.email',
```

Comment:
I think we should add one more layer since confirmation email is a little too generic

`send.signup.confirmation.email`

Reply:
agreed